From 9bbf0a09b75016f5c67ef37dc100157e156edd3e Mon Sep 17 00:00:00 2001
From: Dong Aisheng <aisheng.dong@nxp.com>
Date: Fri, 14 Oct 2016 12:21:19 +0800
Subject: [PATCH 1228/5242] MLK-13340 dts: mx6ul-lpddr2-arm2: fix sd gpio
 polarity

commit  09bda692ffc6c9a718cd9bb468f9bec22d5b3c12 from
https://source.codeaurora.org/external/imx/linux-imx.git

system can't detect SD card due to wrong gpio polarity.

Signed-off-by: Dong Aisheng <aisheng.dong@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx6ul-14x14-lpddr2-arm2.dts |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm/boot/dts/imx6ul-14x14-lpddr2-arm2.dts b/arch/arm/boot/dts/imx6ul-14x14-lpddr2-arm2.dts
index 0692bf4..0ec9509 100644
--- a/arch/arm/boot/dts/imx6ul-14x14-lpddr2-arm2.dts
+++ b/arch/arm/boot/dts/imx6ul-14x14-lpddr2-arm2.dts
@@ -340,8 +340,8 @@
 	pinctrl-0 = <&pinctrl_usdhc1>;
 	pinctrl-1 = <&pinctrl_usdhc1_100mhz>;
 	pinctrl-2 = <&pinctrl_usdhc1_200mhz>;
-	cd-gpios = <&gpio4 26 0>;
-	wp-gpios = <&gpio4 25 0>;
+	cd-gpios = <&gpio4 26 GPIO_ACTIVE_LOW>;
+	wp-gpios = <&gpio4 25 GPIO_ACTIVE_HIGH>;
 	keep-power-in-suspend;
 	enable-sdio-wakeup;
 	vmmc-supply = <&reg_sd1_vmmc>;
-- 
1.7.9.5

